middleware.go

```diff
@@ -24,6 +24,9 @@ import (
 
 	"github.com/ant0ine/go-json-rest/rest"
 	"github.com/mendersoftware/deployments/config"
+	"github.com/mendersoftware/deployments/utils/accesslog"
```

Comment:
ok, but let's handle it in a separate PR, current TODOs are:
1. fixes for comments so far
2. reqid handling (a lot is done, but still in progress)
3. refactoring to lib

2 and 3 are orthogonal, so I suggest we close this PR after 1 is handled to avoid messy converstations on unrelated subjects.